coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -175,6 +175,7 @@ RDEPEND="${RDEPEND}
 	sys-block/parted
 	sys-boot/efibootmgr
 	sys-cluster/ipvsadm
+	sys-devel/gettext
```

Comment:
I'd say it's not obvious why we would want gettext to be in production images, so could you add a comment about it being added for `envsubst` binary that people found useful?